setup.py

```diff
@@ -6,26 +6,26 @@
 from setuptools import Command, setup
 
 
-VENDOR_PATH = 'src/filebrowser_upload/vendor'
+VENDOR_PATH = "src/filebrowser_upload/vendor"
```

Comment:
I prefer config black to not change string quotes, it cause too many changes.

Reply:
Restored single quotes. There was a mixed situation with single and double quotes, that's why I picked double quotes.